news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,81 @@
+  
+## Release schedule
+
+- ☑ a bunch of checkboxes.
+  
+## Summary of changes
+
+### Stamina
+
+The old stamina system assumed a two-finger KDDK full-alt playstyle, with each hand being calculated separately, while the new stamina system calculates the theoretical minimum stamina required for any pattern — the assumption is that notes of a particular colour are always alternated.
+
+This does not accurately represent any single playstyle, but is desirable because we cannot feasibly detect playstyles. This "minimum" of sorts is therefore the fairest judgement of difficulty — any additional difficulty arising from specific patterning is attached to the playstyle itself.
+
+### Colour
+
+The old colour system detected the frequency of colour changes and had penalties for specific repeated patterns, while the new colour system evaluates all patterns using a compression algorithm, and adds those patterns to a dictionary.
+
+The idea is that the more information that is needed to describe a pattern, the harder it is. Therefore, maps with a large variety of patterns (large pattern dictionary in the code) will be evaluated as being harder.
+
+This dictionary also has a "memory" of the patterns used. If a known pattern reappears later in the map, its difficulty will scale according to how long ago it last appeared in the map, which means repeated patterns close together are nerfed.
+
+### PP rework
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values (around 40% lower). In addition, the survey indicated that the way performance points are calculated needed to be re-evaluated. Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance.
+
+#### Global changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which allows for balance changes between different types of scores. The following global multipliers have been adjusted:
+
+- The global multiplier has been increased by **1.13x**, affecting all scores.
+- The HD global multiplier has been further increased by **1.05x**, to increase the gap between EZHD- and HD-only scores
+- An EZ global multiplier of **0.975x** has been added, to increase the gap between EZ and NM scores
+
+In addition, an *effective miss count* has been introduced — this is calculated as `misses * 1000 / successful hits`, which increases the penalty of misses on maps below 1000 max combo and vice versa.
+
+#### Difficulty
+
+The following changes should result in impressive scores on harder beatmaps rewarding more pp.
+
+- The miss penalty has been reduced by 1%. (This is part of an exponential term, so the effect is significantly more than this.)
+- The impact accuracy has on difficulty pp has been increased by 2x.
```

Comment:
this is not a "2x" increase in impact

https://github.com/ppy/osu/compare/f8830c6850128456266c82de83273204f8b74ac0...master#diff-6debfbfde9d2308d19399526b0e127645ac5399d6fb5527c3ef5ef53c0d435c1

![image](https://user-images.githubusercontent.com/36758269/187051731-78fdbabb-68c8-4eaa-bdd5-86acdc83673a.png)

difficulty pp now decreases by the square of decreasing accuracy instead of linearly, which is a huge nerf

![image](https://user-images.githubusercontent.com/36758269/187051879-d6c33fe0-f4a4-4e72-a49b-9d7611522c1e.png)